.github/workflows/publish_container.yml

```
@@ -2,16 +2,16 @@ name: Publish Docker
 on:
   push:
     branches:
-      - master
+      - main
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
+    - uses: actions/checkout@main
     - name: Publish to Registry
-      uses: elgohr/Publish-Docker-Github-Action@master
+      uses: elgohr/Publish-Docker-Github-Action@main
```

I don't think you're supposed to change that one.
```suggestion
      uses: elgohr/Publish-Docker-Github-Action@master
```

---

Agreed